docs/eventing/samples/sequence/sequence-reply-to-event-display/README.md

```diff
@@ -0,0 +1,194 @@
+---
+title: "Sequence Wired to event-display"
+weight: 20
+type: "docs"
+---
+
+# Using Sequences in series
```

Comment:
I could add a backpointer to it, however, my thinking was that the landing page will be sequence.md and then the links point to specific examples like this.
Perhaps, I should switch Prerequisites with the Overview and then from Overview add the link to Sequence (../../../../sequence.md)?

Reply:
@vaikas-google It's very hard to guarantee a particular reading path (thanks to Google search, people sharing links in Slack, people's own preferences when browsing, etc.) so it's best to add links to related content so that people can find the information no matter where they land.

I would be in favor of moving "Overview" up before "Prerequisites" and adding a link to the sequence page there.

Reply:
done.